### ISSUE TITLE
NPCs refuse to be fingered

### ISSUE THREAD

**Issue by A-Master-of-Puppets** [Innoxia/liliths-throne-public#247]
I started a new game, and when I got to Alexandria I thought I'd finger her for a bit before fucking her. Apparently she had other ideas, as she would consistently stop me from doing so _every single time._ I believe this is caused by the fix for the issue where NPCs would _only_ want to finger, but it goes too far in the other direction. NPCs will also consistently start penetration as soon as they can, rather than waiting for the PC to do so. All of this would be fine if the NPC is the dominant one, but all of this is happening when the PC is supposedly the dominant one.

**Comment by TadUnlikely** [comment]
I don't believe this is an issue any longer in newer builds. If @A-Master-of-Puppets can't confirm, I'll see about testing it soon.
